### DISCUSSION TITLE
Is solving IK for multiple end-effectors supported? e.g. with a multi-finger hand?

### DISCUSSION THREAD

**Discussion by zhouxian** [NVlabs/curobo#81] [Software Q&A] [answered]
Hi, thanks for sharing your amazing work. It looks like for IK solving, the current config only allows one single ee_link. In case of using multi-finger robot, we have one single base and potentially multiple end-effectors. Is these case supported or will be supported in the future?
Thanks.

**Comment by balakumar-s** [comment, marked as answer]
Yes, it's supported  in the latest release. You need to add the additional links to track as `link_names:[link_name, link_name_2]` in your robot configuration file. Check this file for an example: https://github.com/NVlabs/curobo/blob/main/src/curobo/content/configs/robot/iiwa_allegro.yml

Once you set the link names, you can set the targets for these frames by send them as a list of Pose for this argument: https://github.com/NVlabs/curobo/blob/1458f79a316855efde6ad314d5cffb20f10aecf0/src/curobo/wrap/reacher/ik_solver.py#L456
